From 5280a91817a6e5b487a547ca2d22aaaa8a873f85 Mon Sep 17 00:00:00 2001
From: Wang Han <416810799@qq.com>
Date: Wed, 1 Aug 2018 20:58:10 +0800
Subject: [PATCH 3/5] Revert "[3/3] cmsdk: add burnIn protection setting"

* Our overlay name now confuses people as Google introduced a screen
  burnin protection feature for ambient mode in Oreo. Just re-use AOSP
  one and kill this.

This reverts commit 088ddf5aafbb578ae5566bec3fab580d1372f76.

Change-Id: Ia76b28444cd7b117559062fbd8230602962616b2
---
 api/lineage_current.txt                              |  1 -
 lineage/res/res/values/config.xml                    |  3 ---
 lineage/res/res/values/symbols.xml                   |  3 ---
 .../java/lineageos/providers/LineageSettings.java    | 12 ------------
 4 files changed, 19 deletions(-)

diff --git a/api/lineage_current.txt b/api/lineage_current.txt
index 8ab220b..cb4b518 100644
--- a/api/lineage_current.txt
+++ b/api/lineage_current.txt
@@ -897,7 +897,6 @@ package lineageos.providers {
     field public static final java.lang.String STATUS_BAR_SHOW_WEATHER = "status_bar_show_weather";
     field public static final java.lang.String STYLUS_ICON_ENABLED = "stylus_icon_enabled";
     field public static final java.lang.String SWAP_VOLUME_KEYS_ON_ROTATION = "swap_volume_keys_on_rotation";
-    field public static final java.lang.String SYSTEMUI_BURNIN_PROTECTION = "systemui_burnin_protection";
     field public static final java.lang.String SYSTEM_PROFILES_ENABLED = "system_profiles_enabled";
     field public static final java.lang.String SYS_PROP_LINEAGE_SETTING_VERSION = "sys.lineage_settings_system_version";
     field public static final java.lang.String T9_SEARCH_INPUT_LOCALE = "t9_search_input_locale";
diff --git a/lineage/res/res/values/config.xml b/lineage/res/res/values/config.xml
index d84aef9..9122403 100644
--- a/lineage/res/res/values/config.xml
+++ b/lineage/res/res/values/config.xml
@@ -22,9 +22,6 @@
     <integer name="config_proximityCheckTimeout">250</integer>
     <bool name="config_proximityCheckOnWakeEnabledByDefault">false</bool>
 
-    <!-- BurnIn protection. This should be enabled on devices that have OLED displays -->
-    <bool name="config_enableBurnInProtection">false</bool>
-
     <!-- Default values for LiveDisplay -->
     <integer name="config_dayColorTemperature">6500</integer>
     <integer name="config_nightColorTemperature">4800</integer>
diff --git a/lineage/res/res/values/symbols.xml b/lineage/res/res/values/symbols.xml
index e61ee90..02ba9ae 100644
--- a/lineage/res/res/values/symbols.xml
+++ b/lineage/res/res/values/symbols.xml
@@ -111,9 +111,6 @@
     <!-- Camera key type -->
     <java-symbol type="bool" name="config_singleStageCameraKey" />
 
-    <!-- BurnIn protection -->
-    <java-symbol type="bool" name="config_enableBurnInProtection" />
-
     <!-- Notification light mappings -->
     <java-symbol type="array" name="notification_light_package_mapping" />
 
diff --git a/sdk/src/java/lineageos/providers/LineageSettings.java b/sdk/src/java/lineageos/providers/LineageSettings.java
index 17a81c9..f9bd057 100644
--- a/sdk/src/java/lineageos/providers/LineageSettings.java
+++ b/sdk/src/java/lineageos/providers/LineageSettings.java
@@ -1695,17 +1695,6 @@ public final class LineageSettings {
         public static final Validator STATUS_BAR_BRIGHTNESS_CONTROL_VALIDATOR =
                 sBooleanValidator;
 
-
-        /**
-         * Whether to enable burnin protection
-         0 = 0ff, 1 = on
-         */
-        public static final String SYSTEMUI_BURNIN_PROTECTION = "systemui_burnin_protection";
-
-        /** @hide */
-        public static final Validator SYSTEMUI_BURNIN_PROTECTION_VALIDATOR =
-                sBooleanValidator;
-
         /**
          * Whether or not volume button music controls should be enabled to seek media tracks
          * 0 = 0ff, 1 = on
@@ -2293,7 +2282,6 @@ public final class LineageSettings {
             VALIDATORS.put(QS_SHOW_BRIGHTNESS_SLIDER, QS_SHOW_BRIGHTNESS_SLIDER_VALIDATOR);
             VALIDATORS.put(STATUS_BAR_BRIGHTNESS_CONTROL,
                     STATUS_BAR_BRIGHTNESS_CONTROL_VALIDATOR);
-            VALIDATORS.put(SYSTEMUI_BURNIN_PROTECTION, SYSTEMUI_BURNIN_PROTECTION_VALIDATOR);
             VALIDATORS.put(VOLBTN_MUSIC_CONTROLS, VOLBTN_MUSIC_CONTROLS_VALIDATOR);
             VALIDATORS.put(USE_EDGE_SERVICE_FOR_GESTURES,
                     USE_EDGE_SERVICE_FOR_GESTURES_VALIDATOR);
-- 
2.17.1

